src/messaging.js

```diff
@@ -76,8 +70,23 @@ export default class Messaging {
     this.localMessaging.broadcastMessage({topic: 'licensing-watch', data: {component_name: this.config.componentName, component_id: this.componentId}});
   }
 
+  requestTwitterStatusRequest(message) {
+    let running = false;
+
+    const clients = message.clients;
+    const required = ['twitter'];
+
+    running = required.every((val) => clients.includes(val));
```

Comment:
@jungeunyoon are we expecting to require something else that twitter here ? if not these lines could be simplified to clients.includes( 'twitter' )

Reply:
@santiagonoguez sure, will change it 👍